README.md

```diff
@@ -35,38 +35,36 @@ We'll implement the tool in several incremental phases to get some basic functio
  
 Based on Tim's feedback, instead of using `<xsdl:include/>` as originally proposed, change it so that it can be used to either include _all_ the scripts in the referenced file:
 
-	<Binding>
-		<Definition>
-			<Script>
-				<Transforms>
-					<xsdl:EspRequest>
-						<xsdl:include-from file="service-scripts.xml" all="true"/>
-						<!-- 
-								All the transforms defined in service-scripts.xml
-								are included here in this element
-						-->
-					</xsdl:EspRequest>
-				</Transforms>
-			</Script>
-		</Definition>	
-	</Binding>
+  <Binding>
+    <Definition>
+      <Script>
+        <Transforms>
+          <EspRequest>
+            <include file="service-scripts.xml"/>
+		<!-- 
+			By default all the transforms defined in service-scripts.xml
+			are included here in this element
+		-->
+          </EspRequest>
+        </Transforms>
+      </Script>
+    </Definition>	
+  </Binding>
 
 or just include some by name:
 
-	<Binding>
-		<Definition>
-			<Script>
-				<Transforms>
-					<xsdl:EspRequest>
-						<xsdl:include-from file="service-scripts.xml" all="false">
-							<xsdl:include transform="GenericOutOfBand"/>
-							<xsdl:include transform="UserOutOfBand"/>
-						</xsdl:include-from>
-					</xsdl:EspRequest>
-				</Transforms>
-			</Script>
-		</Definition>	
-	</Binding>
+  <Binding>
+    <Definition>
+      <Script>
+        <Transforms>
+          <EspRequest>
+            <include file="service-scripts.xml" transform="GenericOutOfBand"/>
+            <include file="service-scripts.xml" transform="UserOutOfBand"/>
```

Comment:
I can go either way on including one or multiple named transforms with a single statement, although if multiple are allowed the attribute name would need to reflect the possibility of multiple. Because statement order matters, we have to be able to reference a file multiple times (include from A, then B, then A again, so why not A, A, B, A). On the other hand, including multiple with one statement could simplify markup without really complicating the implementation.